patterns/newtype.md

```diff
@@ -97,6 +100,7 @@ Here, `Bar` might be some public, generic type and `T1` and `T2` are some intern
 
 ## See also
 
+- [Advanced Types in the book](https://doc.rust-lang.org/book/ch19-04-advanced-types.html?highlight=newtype#using-the-newtype-pattern-for-type-safety-and-abstraction)
 - [Newtypes in the style guide](https://doc.rust-lang.org/1.0.0/style/features/types/newtype.html)
```

Comment:
The style guide is outdated or not maintained IIRC.

Reply:
Sad, but true.

```suggestion
```